lib/util/cache/global/redis.ts

```diff
@@ -22,7 +22,13 @@ async function rm(namespace: string, key: string): Promise<void> {
   await client?.del(getKey(namespace, key));
 }
 
-async function get<T = never>(namespace: string, key: string): Promise<T> {
+export async function get<T = never>(
+  namespace: string,
+  key: string
+): Promise<T> {
+  if (!client) {
```

Comment:
We don't need this check, as we use the question operator later.

Reply:
Let's go with explicit rather than clever. I'll remove the later check